File: Server/runes/SetHeroHealth.js
var entities = require('../entityManager');


exports.execute = function(rune, state) {
       var controller = entities.getEntity(rune.heroGuid, state);
       controller.hero.health = rune.health;
}

exports.CreateCard = function(heroGuid, health)
{
    var rune = {
        "runeType":"SetHeroHealth",
        "heroGuid":heroGuid,
        "health":health
    }
    
    return rune;
}

exports.canSee = function (params) {
    return true;
} 

File: Assets/Scripts/Runes/SetHeroHealth.cs
using UnityEngine;
using System.Collections;
using System;

public class SetHeroHealth : Rune {

    public string hero_uid { get; set; }
    public int health { get; set; }

    public override void Execute(Action action)
    {
        action();
    }

    public override void OnGUI()
    {

    }

}
